Cargo.toml

```diff
@@ -19,3 +19,5 @@ tee = "0.1.0"
 toml = "0.5.6"
 clap = { version = "4.0.15", features = ["derive", "wrap_help"] }
 shlex = "1.1.0"
+image = { version = "0.24.7", default-features = false, features = ["png"] }
```

Comment:
relating to my other comment, we could also consider [`lodepng`](https://lib.rs/crates/lodepng) (`smdhtool` uses the C++ version of this I think) or [`png`](https://lib.rs/crates/png) as potentially lighter-weight alternatives, although `image` is nice in that we could potentially support other image formats. I'm not sure that we really need to support anything other than PNG though honestly, it's a pretty universal format that most users should be able to produce.

@Meziu any preference here?

Reply:
`image` without all of its default features doesn't have many dependencies, and with `png` feature it uses `png` crate under the hood

Reply:
Ah, I didn't realize that `cytryna` itself had a direct dependency on `image`. It should be fine though, as you said its dependency tree is not too large